src/App.svelte

```diff
@@ -1,182 +1,268 @@
 <script lang="ts">
-    import { onMount } from 'svelte';
-    import { now } from 'svelte/internal';
+  import { onMount } from "svelte";
+  import { now } from "svelte/internal";
 
-    import Content from './Content.svelte';
-    import Message from './Message.svelte';
-    import SSOButton from './SOOButton.svelte';
-    import LogoutButton from './LogoutButton.svelte';
-    import SOORevokeButton from './SOORevokeButton.svelte';
-    import { validate } from './session';
+  import Modal from "./Modal.svelte";
```

Comment:
Are you using prettier?

Reply:
I used a different formatter maybe. Now, the code is updated according to the main prettierrc file. @glaslos